content/chronograf/v1.9/guides/advanced-kapacitor.md

```diff
@@ -1,73 +1,77 @@
+- [Manage Kapacitor tasks](#manage-kapacitor-tasks)
 
+## Manage Kapacitor alerts
 
-Chronograf provides a user interface for [Kapacitor](/{{< latest "kapacitor" >}}/), InfluxData's processing framework for creating alerts, running ETL jobs, and detecting anomalies in your data.
-This guide offers insights into how Kapacitor interacts with Chronograf and introduces advanced Kapacitor usage within Chronograf.
+Chronograf provides information about Kapacitor alerts on the Alert History page.
+Chronograf writes Kapacitor alert data to InfluxDB as time series data.
+It stores the data in the `alerts` [measurement](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#measurement)
+in the `chronograf` database.
+By default, this data is subject to an infinite [retention policy](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp) (RP).
+If you expect to have a large number of alerts or do not want to store your alert
+history forever, consider shortening the [duration](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#duration)
+of the default retention policy.
 
+### Modify the retention policy of the chronograf database
 
-## Alert history management
-
-Chronograf stores the information on the Alert History page as time series data in InfluxDB.
-It stores it in the `chronograf` database and in the `alerts` [measurement](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#measurement).
-By default, this data is subject to an infinite [retention policy](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp) (RP), that is, InfluxDB stores them forever.
-Users who expect to have a large number of alerts and users who do not want to store their alert history forever may want to shorten the [duration](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#duration) of that retention policy.
-
-### Modifying the retention policy in Chronograf
-
-Use the Chronograf Admin page to modify the retention policy in the `chronograf` database.
+Use the Chronograf **Admin page** to modify the retention policy in the `chronograf` database.
 In the Databases tab:
 
-#### Step 1: Locate the `chronograf` database and click on the infinity symbol (∞)
-
-![RP in practice](/img/chronograf/1-6-g-advkap-dur.png)
-
-#### Step 2: Enter a different duration
-
-The minimum allowable duration is one hour (`1h`) and the maximum is infinite (`INF`).
-See the InfluxDB documentation for the list of [acceptable duration units](/{{< latest "influxdb" "v1" >}}/query_language/spec/#duration-units).
-
-#### Step 3: Click the green check mark to save your changes
-
-InfluxDB only keeps data in the `chronograf` database that fall within that new duration; the system automatically deletes any data with timestamps that occur before the duration setting.
+1.  Click **{{< icon "crown" >}} InfluxDB Admin** in the left navigation bar.
+2.  Hover over the retention policy list of the `chronograf` and click **Edit**
```

Comment:
Is there a word missing here? "of the `chronograf` something"?

Reply:
Yep, should say "`chronograf` database." Fixed it.